crates/ruff_python_ast/src/helpers.rs

```diff
@@ -111,6 +111,7 @@ where
                 | Expr::Subscript(_)
                 | Expr::Yield(_)
                 | Expr::YieldFrom(_)
+                | Expr::LineMagic(_)
```

Comment:
I believe there have been a couple of PRs that all required the same change. Is there a more sustainable approach to this than fixing them one by one?

Reply:
Correct me if I'm wrong but I think this is defined in a helper function itself (`contains_effect`) or are you talking about something else? I'll merge this for now.